pkg/conversion/convert.go

```diff
@@ -22,11 +22,13 @@ type analyzerRules struct {
 	relativePath string
 }
 
-func ConvertWindupRulesetsToAnalyzer(windups []windup.Ruleset, baseLocation, outputDir string, flattenRulesets bool) (map[string]*analyzerRules, error) {
-	// Write discovery rules
-	err := writeDiscoveryRules(outputDir)
-	if err != nil {
-		return nil, err
+func ConvertWindupRulesetsToAnalyzer(windups []windup.Ruleset, baseLocation, outputDir string, flattenRulesets bool, cmd string) (map[string]*analyzerRules, error) {
```

Comment:
Could we just make this argument something like `writeDiscoveryRules`  rather than making it aware of the subcommand running it?